### DISCUSSION TITLE
Go module updates not being detected? What am I missing here?

### DISCUSSION THREAD

**Discussion by loganmarchione** [renovatebot/renovate#19334] [Request Help] [answered]
I have a Hugo blog located [here](https://github.com/loganmarchione/loganmarchione.com) (Hugo is written in Go and uses Go modules). My renovate.json file is [here](https://github.com/loganmarchione/loganmarchione.com/blob/master/.github/renovate.json).

My `go.mod` file looks like this.

```
module github.com/loganmarchione/loganmarchione.com

go 1.19

require (
	github.com/adityatelange/hugo-PaperMod v0.0.0-20220717061433-87ffee9fcf6d // indirect
	github.com/foo-dogsquared/hugo-mod-simple-icons v0.0.0-20220706232029-07de82f7804d // indirect
)
```

Reading [this page](https://docs.renovatebot.com/golang/), I have the `require` statement and the `v0.0.0-` syntax, but Renovate isn't picking up on the newer versions of modules. 

For example, in [github.com/foo-dogsquared/hugo-mod-simple-icons](https://github.com/foo-dogsquared/hugo-mod-simple-icons/commits/master), the current hash `07de82f` is from July 2022, but the latest `ee9d5f5` is from December 2022.

I'm seeing this in the logs, so it doesn't even look like it's detecting the modules. What am I doing wrong?
```
    "gomod": [
      {
        "packageFile": "go.mod",
        "deps": [
          {
            "managerData": {
              "lineNumber": 2
            },
            "depName": "go",
            "depType": "golang",
            "currentValue": "1.19",
            "datasource": "golang-version",
            "versioning": "npm",
            "rangeStrategy": "replace",
            "depIndex": 0,
            "updates": [],
            "warnings": [],
            "sourceUrl": "https://github.com/golang/go",
            "homepage": "https://go.dev/",
            "currentVersion": "1.19.4"
          }
        ]
      }
    ]
```

**Comment by rarkins** [comment, marked as answer]
right now indirect dependencies are ignored

**Reply by loganmarchione** [reply]
Ahh ok, thanks! Is there a way I can override this?

**Reply by rarkins** [reply]
No, but it's planned
